src/portgraph.rs

```diff
@@ -303,10 +303,9 @@ impl PortGraph {
     ///
     /// # Errors
     ///
-    ///  - When `port_from` or `port_to` does not exist.
-    ///  - When `port_from` is not an output port.
-    ///  - When `port_to` is not an input port.
-    ///  - When `port_from` or `port_to` is already linked.
+    ///  - If `port_from` or `port_to` does not exist.
```

Comment:
If the edge goes in either direction, then let's call them `a` and `b` rather than `to` and `from`. (I think the terms are unambiguous, From == Outgoing, To == Incoming)